### ISSUE TITLE
get_graph attribute not found

### ISSUE THREAD

**Issue by UltraHector** [pytransitions/transitions#343]
It seems that if you use the LockedHierarchicalGraphMachine and use 'add_model' method. 
The get_graph method is not added to the models. The following code will reproduce the error.
```
from transitions.extensions import LockedHierarchicalGraphMachine
class Matter():
    pass

lump1 = Matter()
lump2 = Matter()

states=['solid', 'liquid', 'gas', 'plasma']
transitions = [
    { 'trigger': 'melt', 'source': 'solid', 'dest': 'liquid'},
]

machine = LockedHierarchicalGraphMachine(states=states, transitions=transitions, initial='solid')

machine.add_model(lump1)
machine.add_model(lump2, initial='solid')

lump2.melt()
print(lump1.state)
```

**Comment by aleneum** [comment]
Hi @UltraHector,

that's a bug! `GraphMachine` had no `add_model` method which means that only initially passed models were decorated with a `get_graph` method. I added a test and fixed the issue for `GraphMachine` for the next release.

 Sorry for the long delay and thank you for the report!

Best,
Alex

**Comment by aleneum** [comment]
This has been solved by 258f5b1. A pypi version containing the fix has been released. I will close that issue for now. If you still encounter that bug, please comment again and I will reopen this issue.
